modules/geo-layers/src/tile-layer/tile-2d-header.js

```diff
@@ -1,13 +1,17 @@
 import {log} from '@deck.gl/core';
+import RequestScheduler from '@loaders.gl/loader-utils';
```

Comment:
`{RequestScheduler}` ?

Reply:
Yes, during testing I copied `request-scheduler.js` into the current directory and was using `import RequestScheduler from './request-scheduler.js'`. I'm not clear on the best way to otherwise test against `loaders.gl` master from within `deck.gl`.

Reply:
Do you use `yarn link` across repos?

Reply:
- I did publish loaders.gl@2.2.0-alpha.1 from the latest master. You can probably force it with a "resolutions" field in your package.json.

- Also I think the older version of the request scheduler is exported with underscore from loaders.gl/loader-utils version that is used by deck.gl